manifests/config.pp

```diff
@@ -41,7 +41,7 @@
   foreman_proxy::settings_file { 'tftp': }
   foreman_proxy::settings_file { 'realm': }
 
-  if $foreman_proxy::puppetca or $foreman_proxy::puppetrun {
+  if $foreman_proxy::puppetca or $foreman_proxy::puppetrun or tagged('foreman_proxy::plugin::salt') {
```

Comment:
What about a separate boolean in `foreman_proxy/init.pp` for managing Salt sudo rules? It means two options to pass to the installer but it could be worse...